packages/components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -4,486 +4,216 @@ import React, {
-                        rightMargin={horizontalPadding}
-                        onClick={() => {
-                            if (masterLink.callback) {
-                                masterLink.callback();
-                                setToggled(false);
-                            }
-                        }}
-                    >
-                        <span>{masterLink.label}</span>
-                        <MasterLinkComponentIcon
-                            icon={masterLink.icon}
-                            size={10}
-                            color={theme.TYPE_GREEN}
-                        />
-                    </MasterLinkComponent>
-                )}
-                {visibleItems.map(group => (
-                    <React.Fragment key={group.key}>
-                        {group.label && <Group>{group.label}</Group>}
-                        {group.options.map(item => (
-                            <MenuItemComponent
-                                onClick={e => {
-                                    e.stopPropagation();
-                                    onMenuItemClick(item);
-                                }}
-                                data-test={item['data-test']}
-                                key={item.key}
-                                item={item}
-                            >
-                                {getIconComponent(item)}
-                                <MenuItemLabel>{item.label}</MenuItemLabel>
-                                {item.iconRight && (
-                                    <IconRight>
-                                        <Icon
-                                            icon={item.iconRight}
-                                            size={16}
-                                            color={theme.TYPE_DARK_GREY}
-                                        />
-                                    </IconRight>
-                                )}
-                            </MenuItemComponent>
-                        ))}
-                    </React.Fragment>
-                ))}
-            </MenuComponent>
+                addon={addon}
+            />,
+            document.body,
         );
 
-        const portalMenu = absolutePosition && appendTo ? createPortal(menu, appendTo) : menu;
-
         return (
-            <Wrapper className={className} absolutePosition={!!absolutePosition}>
-                {toggleComponent}
-                {toggled && portalMenu}
-            </Wrapper>
+            <Container
+                ref={toggleRef}
+                className={className}
+                type="button"
+                tabIndex={renderOnClickPosition ? -1 : 0}
+                disabled={isDisabled}
+                onClick={onToggleClick}
+                onFocus={() => !isDisabled && !renderOnClickPosition && setToggled(true)}
+                onBlur={e => !menuRef.current?.contains(e.relatedTarget) && setToggled(false)}
```

Comment:
Am I right the condition shouldn't be here anymore after changing how Tab works? Otherwise you have to press Tab three times to close the second Dropdown in Storybook.

Reply:
https://github.com/trezor/trezor-suite/pull/9350/commits/58a53e5cfd3ba1fb33e45822432019514f139c02